mysql-test/suite/rocksdb/t/add_index_inplace.test

```diff
@@ -342,4 +342,28 @@ while ($i <= $max) {
 
 DROP TABLE t1;
 
+# Test that fast secondary index creation updates cardinality properly
+CREATE TABLE t1 (i INT PRIMARY KEY, j INT) ENGINE = ROCKSDB;
+INSERT INTO t1 VALUES (1,2), (2,4), (3,6), (4,8), (5,10);
+ALTER TABLE t1 ADD INDEX kj(j), ALGORITHM=INPLACE;
```

Comment:
as discussed offline, @hermanlee will make a separate diff including this test case to be squashed into this diff later